spec/acceptance/04_mysql_backup_spec.rb

```diff
@@ -129,7 +129,7 @@ class { 'mysql::server::backup':
   end
 
   context 'with xtrabackup enabled' do
-    context 'should work with no errors', if: ((os[:family] == 'debian') || (os[:family] == 'ubuntu' && os[:release] =~ %r{^16\.04|^18\.04}) || (os[:family] == 'redhat' && os[:release].to_i > 7)) do
+    context 'should work with no errors', if: ((os[:family] == 'debian') || (os[:family] == 'ubuntu' && os[:release] =~ %r{^18\.04|^20\.04}) || (os[:family] == 'redhat' && os[:release].to_i > 7)) do
```

Comment:
Since we only support Ubuntu 18.04 and 20.04 we don't really need to designate them specifically.
`os[:family] == 'ubuntu'` on it's own should be fine.

Reply:
👍